### PR TITLE
testgrid - upgrade py2 -> py3

### PR DESCRIPTION
**Pull request by clarketm**
Additional conversions for python2 -> python3 ( #13164 ) migration and targets the `testgrid` directory.

### CODE REVIEW

File: testgrid/conformance/upload_e2e.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
 #
 # Usage: see README.md
 
+from __future__ import print_function
```
Comment (clarketm) [inline testgrid/conformance/upload_e2e.py]: > Note: this is required for pylint: 1.9.4 to understand python3 print function.

Comment (spiffxp) [reply testgrid/conformance/upload_e2e.py]: Can you drop this into the file as a comment?

Comment (clarketm) [reply testgrid/conformance/upload_e2e.py]: Done. 👍

**Comment by clarketm** [comment]
@spiffxp - Updated the source file to include the comment.

**Pull request merged**